src/Resources/app/administration/test/e2e/cypress/integration/plugin-config.spec.js

```diff
@@ -0,0 +1,186 @@
+// / <reference types="Cypress" />
+
+const selector = {
+    configElements: [],
+};
+
+const configElements = [
+    {
+        type: "input",
+        label: {
+            selector: 'label[for="PluginConfig.config.email"]',
+            text: "eMail address",
+        },
+        elementSelector: 'input[name="PluginConfig.config.email"]',
+    },
+    {
+        type: "input-disabled",
+        label: {
+            selector: 'label[for="PluginConfig.config.disabledInputField"]',
+            text: "This input field is disabled",
+        },
+        elementSelector:
+            'input[name="PluginConfig.config.disabledInputField"][disabled="disabled"]',
+    },
+    {
+        type: "single-select",
+        options: ["English smtp", "English pop3"],
+        label: {
+            selector: 'label[for="PluginConfig.config.mailMethod"]',
+            text: "Mail method",
+        },
+        elementSelector: 'div[name="PluginConfig.config.mailMethod"]',
+    },
+    {
+        type: "input",
+        label: {
+            selector: 'label[for="PluginConfig.config.secret"]',
+            text: "Secret token",
+        },
+        elementSelector:
+            'input[name="PluginConfig.config.secret"][type="password"]',
+    },
+    {
+        type: "single-select",
+        options: ["Cool Product"],
+        label: {
+            selector: 'label[for="PluginConfig.config.exampleProduct"]',
+            text: "Choose a product for the plugin configuration",
```

Comment:
```suggestion
            text: "Select a product...",
```
Wenn wir an unseren Konventionen bleiben wollen